recipes/lrez/meta.yaml

```
@@ -1,36 +1,40 @@
-{% set name = "lrez" %}
-{% set version = "1.1" %}
-{% set sha256 = "5daac98361ff45f454e1d24ff33f259958d4c94bb627ee01a4beacd5b959db28" %}
+{% set version = "2.0" %}
+{% set sha256 = "9223e915644915b80d3adbe5f2693d9fde80bda1df6f2a2fd773c134c6e73356" %}
 
 package:
   name: lrez
-  version: {{ version }} 
-
-build:
-  number: 1
+  version: '{{version}}'
 
 source:
-  url: https://github.com/flegeai/LRez/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: "https://github.com/morispi/LRez/releases/download/v{{version}}/LRez.tar.gz"
+  sha256: '{{sha256}}'
+
+build:
+  number: 0
 
 requirements:
   build:
     - make
-    - {{ compiler('cxx') }}
+    - {{ compiler('c')}}
+    - {{ compiler('cxx')}}
     - cmake
+    - wget
   host:
-    - htslib
+    - zlib
+    - boost-cpp
   run:
-    - htslib
-    - python>=3
-    - indexed_gzip
-    
+    - boost-cpp
+
 test:
   commands:
-    - idx_bx_sqlite3.py  -h
+    - LRez -h | grep "LRez"
 
 about:
-  home: https://github.com/flegeai/LRez
-  license: file
-  license_file: LICENSE
-  summary: LRez - tools for manipulating 10X fastq and bam files
+  home: https://github.com/morispi/LRez
+  license: AGPL-3.0
+  license_family: AGPL
```

Best is to use SPDX license expressions like so:
```suggestion
  license: AGPL-3.0-or-later AND MIT
  license_file:
    - LICENSE
    - bamtools/LICENSE
```
or
```suggestion
  license: AGPL-3.0-only AND MIT
  license_file:
    - LICENSE
    - bamtools/LICENSE
```
depending on LRez's license.
The `license_family` field is redundant and of limited capability, so I'd just omit it.

(If any further code is bundled (directly or indirectly via BamTools) you should also include the appropriate license information for it in a similar manner.)